core/org.wso2.carbon.user.core/src/main/java/org/wso2/carbon/user/core/ldap/ReadOnlyLDAPUserStoreManager.java

```diff
@@ -600,24 +601,8 @@ public boolean doAuthenticate(String userName, Object credential) throws UserSto
                                                         Boolean.parseBoolean(property);
 
                                                 if (transformObjectGuidToUuid) {
-                                                    // bytes[0] <-> bytes[3]
-                                                    byte swap = bytes[3];
-                                                    bytes[3] = bytes[0];
-                                                    bytes[0] = swap;
-                                                    // bytes[1] <-> bytes[2]
-                                                    swap = bytes[2];
-                                                    bytes[2] = bytes[1];
-                                                    bytes[1] = swap;
-                                                    // bytes[4] <-> bytes[5]
-                                                    swap = bytes[5];
-                                                    bytes[5] = bytes[4];
-                                                    bytes[4] = swap;
-                                                    // bytes[6] <-> bytes[7]
-                                                    swap = bytes[7];
-                                                    bytes[7] = bytes[6];
-                                                    bytes[6] = swap;
-
-                                                    final java.nio.ByteBuffer bb = java.nio.ByteBuffer.wrap(bytes);
+                                                    final java.nio.ByteBuffer bb = java.nio.ByteBuffer.wrap(swapBytes
```

Comment:
We do not need to have the entire package since this is there in the import packages